homeassistant/components/notify/pushetta.py

```diff
@@ -0,0 +1,63 @@
+"""
+homeassistant.components.notify.pushetta
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+Pushetta platform for notify component.
+
+For more details about this platform, please refer to the documentation at
+https://home-assistant.io/components/notify.pushetta/
+"""
+import logging
+
+from homeassistant.helpers import validate_config
+from homeassistant.components.notify import (
+    DOMAIN, ATTR_TITLE, BaseNotificationService)
+from homeassistant.const import CONF_API_KEY
+
+_LOGGER = logging.getLogger(__name__)
+
+REQUIREMENTS = ['pushetta==1.0.15']
+
+
+def get_service(hass, config):
+    """ Get the Pushetta notification service. """
+
+    from pushetta import Pushetta, exceptions
+
+    if not validate_config({DOMAIN: config},
+                           {DOMAIN: [CONF_API_KEY, 'channel_name']},
+                           _LOGGER):
+        return None
+
+    try:
+        pushetta = Pushetta(config[CONF_API_KEY])
+        pushetta.pushMessage(config['channel_name'], "Home Assistant started")
+    except exceptions.TokenValidationError:
+        _LOGGER.error("Please check your access token")
+        return None
+    except exceptions.ChannelNotFoundError:
+        _LOGGER.exception("Channel '%s' not found", config['channel_name'])
```

Comment:
Let's change this to `_LOGGER.error`.

The difference between `error` and `exception` is that `exception` will print out the full stack trace of the current exception. This can help debugging but in this case is really not necessary as we know exactly the cause of the problem: the channel was not found.